compiler/x/codegen/OMRTreeEvaluator.hpp

```diff
@@ -396,6 +396,10 @@ class OMR_EXTENSIBLE TreeEvaluator: public OMR::TreeEvaluator
                                            TR::CodeGenerator *cg,
                                            TR::Instruction *currentInstruction = NULL);
 
```

Comment:
Can you add Doxygen comments for these new APIs please?

Reply:
These aren't new APIs, which made me realize that they belong to OMR::CodeGenerator. Anyway, I added docs to the parent definition.